tests/test_template_types.py

```diff
@@ -819,6 +819,35 @@ def test_subject_line_gets_replaced():
     assert template.subject == 'Jo'
 
 
+@pytest.mark.parametrize("content, values, expected_count", [
+    ("Content with ((placeholder))", {"placeholder": "something extra"}, 28),
+    ("Content with ((placeholder))", {"placeholder": ""}, 12),
+    ("((placeholder))  ", {"placeholder": "  "}, 0),
+    ("  ", {}, 0),
```

Comment:
Could be worth one additional test case for `("some string", {}, 11)` for the case of no stripping and no placeholder

Reply:
I think it's there at line 825